### PR TITLE
Addonly_active_users parameter

### PR DESCRIPTION
**Pull request by balintbartok**
refs: MBL-13735
affects: Student
release note: Fixed activity stream showing unplublished courses.

test plan: See ticket

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>



<details><summary>:x: Swift lint</summary>


    ❌ /Users/vagrant/git/Core/Core/Notifications/APIActivity.swift:90:28: Trailing Comma Violation: Multi-line collection literals should have trailing commas. (trailing_comma)
    
    
</details>


[MBL-13735](https://instructure.atlassian.net/browse/MBL-13735)

#### Affected Apps: Student

#### Release Note: 
Fixed activity stream showing unplublished courses.
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/71039294/p/304cb58074f98f89f439e0d530333f07">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/71039302/p/22e63e9e8daff312d555c81befb0f570">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/71039252/p/20d347b30da32ace421182dd659ead0c">
    
  </a>
</details>

File: Core/Core/Notifications/APIActivity.swift [hunk 1]
```diff
@@ -86,6 +86,7 @@ public struct GetActivitiesRequest: APIRequestable {
     }
 
     public var query: [APIQueryItem] {
-        [ .perPage(perPage) ]
+        [ .value("only_active_courses", "true"),
+          .perPage(perPage), ]
     }
```
Comment (vargaat) [inline Core/Core/Notifications/APIActivity.swift:91]: How about moving the square brackets next to the curly ones?
```
public var query: [APIQueryItem] {[
    .value("only_active_courses", "true"),
    .perPage(perPage),
]}
```

**Review by vargaat — APPROVED** [review]

**Review by idevainstr — APPROVED** [review]
QA+1

**Comment by vargaat** [comment]
QA+1

**Pull request merged**